elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -169,9 +172,14 @@ public ElideResponse get(String path, MultivaluedMap<String, String> queryParams
             JsonApiDocument jsonApiDoc = new JsonApiDocument();
             RequestScope requestScope = new RequestScope(path, apiVersion, jsonApiDoc,
                     tx, user, queryParams, elideSettings);
-            requestScope.setEntityProjection(new EntityProjectionMaker(elideSettings.getDictionary(),
-                    requestScope).parsePath(path));
+            EntityProjection projection = new EntityProjectionMaker(elideSettings.getDictionary(),
+                    requestScope).parsePath(path);
+            requestScope.setEntityProjection(projection);
             BaseVisitor visitor = new GetVisitor(requestScope);
+
+            //accept and process the query here
```

Comment:
This comment string does not add much value as the two functions names describe exactly this comment.